iceoryx_hoofs/test/moduletests/test_cxx_function_ref.cpp

```diff
@@ -312,7 +312,7 @@ TEST_F(function_refTest, CreationWithFunctionPointerWithComplexTypeArgWorks)
     constexpr auto fp = &returnComplexType;
     function_ref<ComplexType(ComplexType)> sut(fp);
 
-    ComplexType arg{1, 2, 3.3};
+    ComplexType arg{1, 2, 3.3f};
```

Comment:
Literals should always be capitalized.

```suggestion
    ComplexType arg{1, 2, 3.3F};
```